_posts/2025-10-06-phoonnx.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Introducing phoonnx: The Next Generation of Open Voice for OpenVoiceOS"
+excerpt: "Today marks a significant step forward in the OpenVoiceOS journey with the official integration and adoption of phoonnx as our primary Text-to-Speech (TTS) engine."
+coverImage: "/assets/blog/phoonnx/thumb.png"
+date: "2025-10-06T00:00:00.000Z"
+author:
+  name: JarbasAl
+  picture: "https://avatars.githubusercontent.com/u/33701864"
+ogImage:
+  url: "/assets/blog/phoonnx/thumb.png"
+---
+
+# Introducing phoonnx: The Next Generation of Open Voice for OpenVoiceOS
+
+Today marks a significant step forward in the OpenVoiceOS journey with the official adoption of [**phoonnx**](https://github.com/TigreGotico/phoonnx) as our primary Text-to-Speech (TTS) framework. 
+
+This new generation of voices is not just about quality; it's about consistency, efficiency, and fulfilling our mission for truly open, offline-ready voice assistants across the globe.
+
+-----
+
+## New Language: Introducing Basque!
+
+Building on our previous work on [**Making Synthetic Voices From Scratch**](https://blog.openvoiceos.org/posts/2025-06-26-making-synthetic-voices-from-scratch) and our successful [**Arabic TTS Collaboration**](https://blog.openvoiceos.org/posts/2025-10-01-arabic_tts_collaboration), we are excited to announce a new milestone in our language support: the addition of new voices for **Basque (eu-ES)**!
+
+This includes both the male voice (**Miro**) and female voice (**Dii**), furthering our mission to support even low-resource languages that lack open, high-quality TTS options.
```

Comment:
Don't know if this is the place to do it, but I would be interested to know how `Dii` is meant to be pronounced. `Miro` is probably self-explanatory, but for consistency, a recording or IPA guide (or both) might be useful to readers